controls/radspreadsheet/howto/customize-row-column-headers.md

```diff
@@ -62,8 +63,34 @@ That's it. The column heading is changed.
         
 
 >tipYou can download a runnable project of the previous example from our online SDK repository [here](https://github.com/telerik/xaml-sdk), the example is listed as __Spreadsheet / CustomRowAndColumnHeadings__.
+
+## States of the Row and Column Headings
+
+The headings of the rows and columns have different states. You can use these states to apply different styles of the items depending on whether they appear in a selection. This section will describe the possible states of the headings in RadSpreadsheet.
+
+
+* **HeadingState**: It is a property of the RowColumnHeadingBase type. It is an enum type and could have the following values:
```

Comment:
Current Version: It is a property of the RowColumnHeadingBase type. It is an enum type and could have the following values:

Suggestions: A property of type RowColumnHeadingBase that gets or sets the heading state. It is an enum and it could have the following values: